code/modules/mob/living/carbon/xenomorph/abilities/defender/defender_powers.dm

```diff
@@ -19,13 +19,13 @@
 		to_chat(X, SPAN_XENOWARNING("You lower your crest."))
 		X.ability_speed_modifier += speed_debuff
 		X.armor_deflection_buff += armor_buff
-		X.mob_size = MOB_SIZE_BIG //knockback immune
+		ADD_TRAIT(X, TRAIT_KNOCKBACK_IMMUNE, TRAIT_SOURCE_INHERENT) //knockback immune
```

Comment:
this has MANY more effects than just wall breaking such as APC destruction, moving vehicles, etc.

Reply:
I consider it a side effect since the original intent is to add immunity to knock back, but since the original implementation changed the mob size instead of using specific trait

Reply:
I don't and as such you're going to have to change how wall breaking checks for validity rather than the defender's size